.travis.yml

```diff
@@ -1,15 +1,13 @@
 env:
   global:
     - LC_CTYPE=en_US.UTF-8
-before_install:
-  - brew update
-  - brew install swiftlint
 matrix:
   include:
     - os: osx
       language: objective-c
-      osx_image: xcode7.3
+      osx_image: xcode8.3
 script: 
+  - pod repo update
```

Comment:
I tried without `pod repo update` and it works, so I removed it. Probably this is a problem only when JSONMatcher quickly upgrade to a new version of a library, in the initial days probably Travis might not have the pod repo updated to include the newest version.